docs/content/Developer-Tools/Developer-Tools.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Introduction
+permalink: /dev-tools
+category: Developer Tools
+menuOrder: 1
+---
+
+Cube.js comes with a number of developer tools:
+
+* [Cube.js CLI][link-cli]
+* [Cube.js Developer Playground][link-dev-playground]
+
+[link-cli]: /using-the-cubejs-cli
+[link-dev-playground]: /dev-tools/dev-playground
```

Comment:
Perhaps we could skip this page? It doesn't seem to add much that isn't already linked/referenced from the sidebar.

Reply:
Sure! Added a commit.